linux_sql/README.md

```diff
@@ -1,2 +1,87 @@
 # Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
```

Comment:
`git rm linux_sql/.README.md.swo linux_sql/.README.md.swp`
do not commit generated files